meshtastic/interfaces/ble/management_service.py

```
@@ -0,0 +1,368 @@
+TRUST_COMMAND_OUTPUT_MAX_CHARS: int = 200
+TRUST_HEX_BLOB_RE = re.compile(r"\b[0-9A-Fa-f]{16,}\b")
+TRUST_TOKEN_RE = re.compile(r"\b[A-Za-z0-9+/=_-]{40,}\b")
+
+T = TypeVar("T")
+
+
+class BLEManagementCommandsService:
+    """Service helpers for BLE management command paths."""
+
+    @staticmethod
+    def execute_management_command(
```

Ruff (0.15.5)

[warning] Too many branches (14 > 12)

(PLR0912)